README.md

```diff
@@ -11,7 +11,7 @@ A curated list of examples related to actix.
 * [actix-realworld-example-app](https://github.com/fairingrey/actix-realworld-example-app) : Implementation of the RealWorld backend API spec in Actix.
 * [Canduma](https://github.com/clifinger/canduma) : rust authentication server boilerplate
 * [Broker](https://crates.io/crates/broker) : Real-time Zero-Code API Server
-
+* [Toplog](https://toplog.cc/) — A Rust Article/Event/Book Aggregator Powered by Actix-web 2.0 [source](https://github.com/danloh/toplog)
```

Comment:
can we add a line break here before contribute?

Reply:
sorry, my bad